server/root.go

```diff
@@ -76,7 +75,7 @@ func StartRestServer(mainCtx ctx.Context, cdc *codec.Codec, registerRoutesFn fun
 	logger.Info(
 		fmt.Sprintf(
 			"Starting application REST service (chain-id: %q)...",
-			viper.GetString(flags.FlagChainID),
```

Comment:
That actually came to my mind too, but didn't want to touch that repo. Btw, let's close this PR once that sdk is modified since that already fixes the issue.

Reply:
Hey @mrostamii, would you be raising the PR on our cosmos fork ?

Reply:
@marcello33 @Raneet10 Maybe better to change `FlagChainID` to `FlagChain` in both repositories? Tested locally and it was fine, but not sure if it's an accepted strategy.

Reply:
Yeah that sounds good, please proceed @mrostamii

Reply:
Hey @mrostamii any update on this? Also, please change target branch to `develop`. Thank you.

Reply:
Done.
CC @marcello33 @Raneet10

Reply:
@mrostamii  once the cosmos PR is merged, the version in heimdall should be updated with a new release and this field `flags.FlagChainID` (now changed to string "`chain`") needs to be updated to `flags.FlagChain`